webserver/src/main/java/com/mlaide/webserver/model/User.java

```diff
@@ -13,6 +15,7 @@
     private String email;
```

Comment:
Ist auch Pflicht, oder?

Reply:
Nicht auf API-Ebene, da wird sie in den schreibenden Requests nicht verwendet.